src/pages/productListing/ProductList.js

```diff
@@ -0,0 +1,46 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import Card from "../../components/Card/Card";
+
+import React from "react";
+import SideBar from "../../components/SideBar/SideBar";
+import Navbar from "../../components/Navbar/Navbar";
+
+const ProductList = () => {
+  const [products, setProducts] = useState("");
```

Comment:
products should be an empty array I guess, why have you used empty string for it?

Reply:
I think it is better to give an empty string as it is a falsy value, but an empty array is a truthy value. 
But if you do give it an empty array as the initial state ,do change the condition in the ternary operator.